pydundas/rest/file.py

```diff
@@ -0,0 +1,35 @@
+from requests import HTTPError
+
+
+class DundasFileNotFoundError(Exception):
+    pass
+
+
+class FileApi:
+    """Uses the Api to manipulate files."""
+
+    def __init__(self, session=None, api=None):
+        self.session = session
+        self.factory = api
+
+    def getFileInfo(self, parent_dir_id, path, parents=''):
+        """
+        Get recursively a file, from its full path.
```

Comment:
Wording issue. Consider: `Retrieve a file recursively using its full path.`
Or if the `get` is important: `Get a file recursively using its full path.`